client/components/Windows/Help.vue

```diff
@@ -583,13 +583,25 @@
 
 			<div class="help-item">
 				<div class="subject">
-					<code>/kick nick</code>
+					<code>/kick nick [reason]</code>
 				</div>
 				<div class="description">
 					<p>Kick a user from the current channel.</p>
 				</div>
 			</div>
 
+			<div class="help-item">
+				<div class="subject">
+					<code>/kickban nick [reason]</code>
+				</div>
+				<div class="description">
+					<p>
+						Kick and ban (<code>+b</code>) a user from the current channel. Unlike /ban,
+						only user names (and not host masks) can be used.
```

Comment:
s/user names/nick/ please. The term nick has a rather specific meaning. "user name" is a bit more up for interpretation (do you mean nick or realname?)

Reply:
Sure, done!